### DISCUSSION TITLE
How to "run" cloned code ?

### DISCUSSION THREAD

**Discussion by anneranch** [blueman-project/blueman#2150] [Q&A]
OK, I am NOT Python fan , but I like to run cloned code. 
What script / file  should I run ?
I am using PyCharm.

**Comment by infirit** [comment]
I highly recommend you install the blueman package provided by your distribution. See a list here https://repology.org/project/blueman/versions

Why are you even cloning the blueman git repo??

**Reply by anneranch** [reply]
Did you read my post ? 

What script / file should I run ?
( what is the executable script?) 

I want to run my cloned copy and be able top modify the code to suit my needs.

**Comment by infirit** [comment]
Let's keep the discussion to one thread. https://github.com/blueman-project/blueman/discussions/2145
